site/content/user-guide/amqp-adapter.md

```diff
@@ -38,6 +35,34 @@ There is a subtle difference between the *device identifier* (*device-id*) and t
 ### SASL EXTERNAL Authentication
 TODO
 
+## AMQP Command-line Client
+For the purposes of demonstrating the usage of the AMQP adapter, the **Hono CLI Module** contains an AMQP command-line client for interacting with the AMQP adapter. This module does not activate the AMQP client by default and should be activated using the `--spring.profile.active` argument.
```

Comment:
simply remove the last sentence, it will  just confuse readers.

Reply:
ok